src/modules/dispatchers/http.py

```diff
@@ -0,0 +1,47 @@
+import json
+import logging
+import os
+import urllib3
+from urllib3.exceptions import HTTPError
+
+
+class HTTPDispatcher(object):
+    def dispatch(self, report):
+        logging.error('Dispatching report via http')
+        http = urllib3.PoolManager()
+        encoded_data = json.dumps(report).encode('utf-8')
+        dispatchMethod = os.environ.get(
+            'KUBEHUNTER_DISPATCH_METHOD',
```

Comment:
I think this wants to be KUBEHUNTER_DISPATCH_METHOD or KUBEHUNTER_HTTP_DISPATCH_METHOD because it only applies to kube-hunter and not other apps that might be run from the same shell / env

Reply:
@lizrice Agreed, the `KUBEHUNTER` should be in the name. but also the `HTTP`. otherwise it's referring to the stdout/http method, rather than GET/POST